rxbinding/src/androidTest/java/com/jakewharton/rxbinding/RxBindingTestRunner.java

```diff
@@ -0,0 +1,34 @@
+package com.jakewharton.rxbinding;
+
+import android.app.KeyguardManager;
+import android.content.Context;
+import android.os.PowerManager;
+import android.support.test.runner.AndroidJUnitRunner;
+
+import static android.content.Context.KEYGUARD_SERVICE;
+import static android.content.Context.POWER_SERVICE;
+import static android.os.PowerManager.ACQUIRE_CAUSES_WAKEUP;
+import static android.os.PowerManager.FULL_WAKE_LOCK;
+import static android.os.PowerManager.ON_AFTER_RELEASE;
+
+public class RxBindingTestRunner extends AndroidJUnitRunner {
+  @Override public void onStart() {
+    runOnMainSync(new Runnable() {
+      @SuppressWarnings("deprecation") // We don't care about deprecation here.
+      @Override public void run() {
+        Context app = getTargetContext().getApplicationContext();
+
+        String name = RxBindingTestRunner.class.getSimpleName();
+        // Unlock the device so that the tests can input keystrokes.
+        KeyguardManager keyguard = (KeyguardManager) app.getSystemService(KEYGUARD_SERVICE);
+        keyguard.newKeyguardLock(name).disableKeyguard();
+        // Wake up the screen.
+        PowerManager power = (PowerManager) app.getSystemService(POWER_SERVICE);
+        power.newWakeLock(FULL_WAKE_LOCK | ACQUIRE_CAUSES_WAKEUP | ON_AFTER_RELEASE, name)
+            .acquire();
```

Comment:
I think the proper thing to do here is store the wake-lock in an instance variable and release it in `onDestroy`. I noticed this the other day in u2020 but haven't made the change yet.